### PR TITLE
Feature: add Player Full List query, PlayersController, index endpoint, include full specs ; remove versioning from controllers/router

### PR DESCRIPTION
**Pull request by tmtocb**
add Player Full List query, PlayersController, index endpoint, include full specs

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -2,5 +2,10 @@
 
 Rails.application.routes.draw do
   root 'home#index'
+
   resources :seasons, only: %i[create]
+
+  namespace 'players' do
+    get 'full_list', to: 'full_list'
```
Comment (Thevel0per) [inline config/routes.rb:9]: I thought we agreed on RESTful routing

File: app/api/players/services/full_list.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Players
+  module Services
```
Comment (Thevel0per) [inline app/api/players/services/full_list.rb:4]: Why this thing is a service? It looks like a query

Comment (tmtocb) [reply app/api/players/services/full_list.rb]: Fixed.

File: config/routes.rb (hunk 1 above)
Comment (tmtocb) [reply config/routes.rb]: @Thevel0per changed to RESTful format, but we need to rethink that as there will be multiple players list actions coming from this controller, only one can be called index

Comment (Thevel0per) [reply config/routes.rb]: Why not just use params?

Comment (tmtocb) [reply config/routes.rb]: I prefer to use business naming :) just a convention I like. 
Let's use RESTful for now with params, over the time we will rethink if when the scale is bigger.

**Review by Thevel0per — APPROVED** [review]

**Pull request merged**